docs/gettingstarted/index.md

```diff
@@ -6,8 +6,8 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain a source distribution from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded source distribution to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain the MCForge Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
```

Comment:
Mod Development Kit

Reply:
You're right! That reads way better! I've updated both PRs to fix this.